pkg/bloomtokenizer/tokenizer.go

```diff
@@ -0,0 +1,175 @@
+package bloomtokenizer
+
+import (
+	"encoding/binary"
+	"unicode/utf8"
+
+	"github.com/grafana/loki/pkg/logproto"
+)
+
+type Token struct {
```

Comment:
I don't think there's an advantage to a specific token struct, especially if the fields are effectively duplicates (one is just a string cast of the other). It could be better served as `[]byte`

Reply:
do we ever do any unsafe casting? I could maybe see a value here in guaranteeing memory consistency by doing it like this if we are into the unsafe casting game.

Reply:
Shouldn't have to -- these are just generated from logs for immediate bloom filter testing

Reply:
I've removed the string value of the struct.  I've left the Token struct there, it does provide a level of readability, but I'm not married to the idea.